src/Configuration/Config.AzureKeyVault/src/AzureKeyVaultConfigurationExtensions.cs

```diff
@@ -23,14 +23,16 @@ public static class AzureKeyVaultConfigurationExtensions
         /// <param name="vault">The Azure KeyVault uri.</param>
         /// <param name="clientId">The application client id.</param>
         /// <param name="clientSecret">The client secret to use for authentication.</param>
+        /// <param name="reloadOnChange">Whether the configuration should be reloaded if the Azure KeyVault changes.</param>
```

Comment:
Where does one specify the polling interval?

Reply:
I imagine that if the user want's fine tune control over the polling interval then they can use the constructor that expects the configuration source instead of these ones. However we could replace this interface to instead of accepting a boolean `reloadOnChange` it accepts an integer `reloadPollingDelay` which would enable reloading on any integer value > -1.